xml/System.IO.Compression/ZipFileExtensions.xml

```diff
@@ -361,11 +361,44 @@
         <Parameter Name="overwriteFiles" Type="System.Boolean" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destinationDirectoryName">To be added.</param>
-        <param name="overwriteFiles">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="source">The <see cref="T:System.IO.Compression.ZipArchive" /> to extract.</param>
+        <param name="destinationDirectoryName">The path to the directory on the file system. The path is permitted to specify relative or absolute path information. Relative path information is interpreted as relative to the current working directory.</param>
+        <param name="overwriteFiles"><see langword="true" /> to indicate overwrite.</param>
```

Comment:
```suggestion
        <param name="overwriteFiles"><see langword="true" /> to indicate that existing files are to be overwritten; <see langword="false" /> otherwise.</param>
```